pkg/make/builder.go

```diff
@@ -598,7 +598,19 @@ func (b *Builder) BuildCmd(src, appName string) func(context.Context) error {
 			return err
 		}
 
-		if stderr.Len() > 0 {
+		// With the Sonoma-era Xcode binaries, we've started seeing a bunch of spurious warnings. They appear to
+		// _mostly_ effect our developer build process. In the interest in not breaking the build, we ignore them.
+		// https://github.com/golang/go/issues/62597#issuecomment-1733893918 is some of them, and some seem related to the zig cross compiling
+		stderrStr := stderr.String()
+		if os.Getenv("GITHUB_ACTIONS") != "" {
+			stderrStr = strings.ReplaceAll(stderrStr, "ld: warning: ignoring duplicate libraries: '-lobjc'\n", "")
+			stderrStr = strings.ReplaceAll(stderrStr, "# github.com/kolide/launcher/cmd/launcher\n", "")
```

Comment:
```suggestion
			stderrStr = strings.ReplaceAll(stderrStr, fmt.Sprintf("# github.com/kolide/launcher/cmd/%s\n", filepath.Base(src)), "")
```

I build with `make -j2 github-build` locally -- this change ensures that `# github.com/kolide/launcher/cmd/launcher.ext` will also match.

Reply:
Might be simpler just to move it to a regex?

Reply:
🤷‍♀️ whichever you prefer, I'm happy as long as github-build works!

Reply:
I used your code.